llvm-project/stage3.cmake

```diff
@@ -0,0 +1,67 @@
+# Please try to keep these cmake variables alphabetically sorted.
+
+# Enable optimizations, as opposed to a debug build.
+set(CMAKE_BUILD_TYPE "Release" CACHE STRING "")
+
+# Use the sysroot we've been building up.
+set(CMAKE_CXX_FLAGS "--sysroot=/sysroot" CACHE STRING "")
+set(CMAKE_C_FLAGS "--sysroot=/sysroot" CACHE STRING "")
+
+# Statically link resulting executable.
+set(CMAKE_EXE_LINKER_FLAGS "-static -lc++abi" CACHE STRING "")
+
+# Set the default target triple to match the host.
+# TODO: passing in the value of $(clang -print-multiarch) causes failures.
+# It seems that alpine clang's default target triple is x86_64-linux-gnu.
+# Perhaps missing alpine in the triple causes some incompatibility?
```

Comment:
I think this comment can be dropped or at the very least changed, as `clang -print-multiarch` was not the right command, `clang -print-target-triple` is.

Reply:
done PTAL